### PR TITLE
Added discovery support for Trango Apex Lynx OS

### PR DESCRIPTION
**Pull request by metavrs**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

File: includes/definitions/trango.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+os: trango
```
Comment (laf) [inline includes/definitions/trango.yaml]: As discussed, looks like this should be apex-lynx or just lynx.

File: includes/polling/os/trango.inc.php [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<?php
+$version = trim(snmp_get($device, ".1.3.6.1.4.1.5454.1.80.1.1.2.0", "-Oqv", "GIGA-PLUS-MIB"), '"');
```
Comment (laf) [inline includes/polling/os/trango.inc.php]: You are using numerical OIDs here but referencing the MIB, you can either change to using the named OID or drop the MIB name as it won't be used.

Same for the below.

File: includes/polling/os/trango.inc.php [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+$hardware = "Trango " . trim(snmp_get($device, ".1.3.6.1.2.1.1.1.0", "-Oqv", "GIGA-PLUS-MIB"), '"');
```
Comment (laf) [inline includes/polling/os/trango.inc.php]: This variable is used to populate some search fields so I think dropping 'Trango ' should be done.

File: mibs/trango/TRANGO-OrionLynx-MIB-COMPACT.mib [hunk 4]
```diff
@@ -0,0 +1,618 @@
+GIGA-PLUS-MIB DEFINITIONS ::= BEGIN
```
Comment (laf) [inline mibs/trango/TRANGO-OrionLynx-MIB-COMPACT.mib]: You can rename this mib file to be `GIGA-PLUS-MIB DEFINITIONS` so it's obvious what it is.

File: mibs/trango/GIGA-PLUS-MIB-DEFINITIONS.mib [hunk 5]
```diff
@@ -0,0 +1,618 @@
+GIGA-PLUS-MIB DEFINITIONS ::= BEGIN
```
Comment (laf) [inline mibs/trango/GIGA-PLUS-MIB-DEFINITIONS.mib]: Just need to drop the .mib from the end of the filename and that's it :)

Comment (metavrs) [inline mibs/trango/GIGA-PLUS-MIB-DEFINITIONS.mib]: done!

**Comment by metavrs** [comment]
Thank you, sir!  My first OS merge :)

**Pull request merged**